src/plugins/loan-channel-query-map.js

```diff
@@ -157,10 +157,10 @@ export default {
+					queryParams: 'partner=62,175,198,225,229,229,263,274,275,285,292,295,301,311,317,342,358,361,369,376,389,390,412,415,417,431,441,446,449,451,452,458,459,461,466,468,472,482,483,486,489,490,491,492,493,494,496,497,498,500,501,502,503,505,506,508,509,510,511,512,513,514,515,516,517,520,521,523,524,525,529,530,531,532,534,535,536,538,539,540,542,543,544,545,546,548,552,553,555,556,557,558,560,561,563,565,566,567,568,569,570,571,573,575,577,578,579,580,581,583,584,585,586,587,591,592,593,595,597,598,599,600,601,602,603,604,608,610,612,613,614,616,617,618&status=fundRaising&sortBy=popularity&distributionModel=both',
+					fallbackUrl: '/lend?partner=62,175,198,225,229,229,263,274,275,285,292,295,301,311,317,342,358,361,369,376,389,390,412,415,417,431,441,446,449,451,452,458,459,461,466,468,472,482,483,486,489,490,491,492,493,494,496,497,498,500,501,502,503,505,506,508,509,510,511,512,513,514,515,516,517,520,521,523,524,525,529,530,531,532,534,535,536,538,539,540,542,543,544,545,546,548,552,553,555,556,557,558,560,561,563,565,566,567,568,569,570,571,573,575,577,578,579,580,581,583,584,585,586,587,591,592,593,595,597,598,599,600,601,602,603,604,608,610,612,613,614,616,617,618&status=fundRaising&sortBy=popularity&distributionModel=both',
 					flssLoanSearch: {
-						partnerId: [510, 503, 131, 186, 173, 249, 200, 228, 350, 485, 482, 498, 525, 547, 523, 532, 539, 540, 542, 552, 565, 568, 570, 575, 586, 495, 530, 546, 589, 497, 501, 511, 516, 518, 492, 587, 545, 531, 566, 585, 508, 513, 573, 580, 581, 555, 536, 486, 594, 592, 345, 491, 505, 500, 502, 509, 512, 520, 526, 529, 591, 494, 524, 506, 496, 517, 489, 490, 515, 535, 548, 556, 577, 583, 315, 558, 593, 514, 553, 560, 569, 571, 534, 543, 557, 563, 258, 418, 574, 538, 567, 561, 408, 218, 544, 281, 466, 305, 397, 437, 225, 390, 493, 521, 356, 198, 480, 369, 469, 295, 311, 417, 301, 379, 274, 317, 285, 62, 175, 40, 358, 578, 361, 477, 392, 528, 576, 365, 211, 479, 483, 330, 389, 292, 415, 403, 412, 263, 431, 275, 376, 342, 468, 446, 461, 451, 445, 449, 452, 465, 458, 459],
+						partnerId: [62, 175, 198, 225, 229, 229, 263, 274, 275, 285, 292, 295, 301, 311, 317, 342, 358, 361, 369, 376, 389, 390, 412, 415, 417, 431, 441, 446, 449, 451, 452, 458, 459, 461, 466, 468, 472, 482, 483, 486, 489, 490, 491, 492, 493, 494, 496, 497, 498, 500, 501, 502, 503, 505, 506, 508, 509, 510, 511, 512, 513, 514, 515, 516, 517, 520, 521, 523, 524, 525, 529, 530, 531, 532, 534, 535, 536, 538, 539, 540, 542, 543, 544, 545, 546, 548, 552, 553, 555, 556, 557, 558, 560, 561, 563, 565, 566, 567, 568, 569, 570, 571, 573, 575, 577, 578, 579, 580, 581, 583, 584, 585, 586, 587, 591, 592, 593, 595, 597, 598, 599, 600, 601, 602, 603, 604, 608, 610, 612, 613, 614, 616, 617, 618],
```

Comment:
Please refresh these ids from the prod instance of the api

Reply:
Done, thanks for pointing this out!